stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -50,9 +53,15 @@ public AsyncPrettyPrinterRegistry getAsyncPrettyPrinterRegistry() {
     return mAsyncPrettyPrinterRegistry;
   }
 
+  public void setPrettyPrinterRegister(@NonNull AsyncPrettyPrinterRegister register) {
+    mPrettyPrinterRegister = register;
+  }
+
   private final PeersRegisteredListener mTempFileCleanup = new PeersRegisteredListener() {
     @Override
     protected void onFirstPeerRegistered() {
+      mAsyncPrettyPrinterRegistry = new AsyncPrettyPrinterRegistry();
```

Comment:
Reading this a second time this is a bit bizarre. Why is this done every time the first client connects? Should this be done once per process invocation? 

ie something like
```
if (mAsyncPrettyPrinterRegistry == null) { 
  mAsyncPrettyPrinterRegistry = new AsyncPrettyPrinterRegistry();
  mPrettyPrinterRegister.registerPrettyPrinters(mAsyncPrettyPrinterRegistry);
}
```